galaxy/settings/local.py

```diff
@@ -0,0 +1,10 @@
+from .testing import *  # noqa: F401,F403
```

Comment:
Is this file included into commit intentionally?

Reply:
local settings shouldn't get checked in, no. Thanks for the catch.